Gemfile.lock

```diff
@@ -282,8 +282,5 @@ DEPENDENCIES
   webmock
   will_paginate-bootstrap
 
-RUBY VERSION
-   ruby 2.3.0p0
-
```

Comment:
hrm hope this doesn't cause problems, perhaps I should add a specific Ruby version and then rebundle? What version do you guys use?